apps/coa/translations/src/en/fields.json

```diff
@@ -89,5 +89,35 @@
         "hint": "If you legal representative or their deeeails changed"
       }
     }
+  },
+  "identity-type": {
+    "legend": "Which information can you provide?",
+    "options": {
+      "passport": {
+        "label": "Passport number"
+      },
+      "brp": {
+        "label": "Biometric residence permit (BRP) number",
+        "hint": "This is on the BRP card or on any letters we have sent"
```

Comment:
content for the hint is a bit different to the figma, the figma is "This is on your BRP card or on any letters we've sent you"